From d953f63dc3794f35148d37c8e4a1e1525ac6f6e2 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:10:13 -0700
Subject: [PATCH] Docs/mm/damon/design: add a link to DAMON sysfs interface
 usage doc

Readers of the design document's 'execution model and data structures'
would wonder how they can configure and start/stop DAMON from the
user-space.  The answer is DAMON sysfs interface.  Add a link to the
DAMON sysfs interface usage document on the design doc for easier
browsing.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index 39e686c6862d..e5591f909ebf 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -16,6 +16,9 @@ called DAMON ``context``.  DAMON executes each context with a kernel thread
 called ``kdamond``.  Multiple kdamonds could run in parallel, for different
 types of monitoring.
 
+To know how user-space can do the configurations and start/stop DAMON, refer to
+:ref:`DAMON sysfs interface <sysfs_interface>` documentation.
+
 
 Overall Architecture
 ====================
-- 
2.39.2

